Sources/TuistCore/Models/TargetAction.swift

```diff
@@ -22,6 +22,11 @@ public struct TargetAction: Equatable {
     /// Path to the script to execute
     public let path: AbsolutePath?
 
+    /// We want to skip linting for actions we create
+    /// Eg when a mapper returns a file descriptor to create a new file
+    /// It will not exist before the initial linting
+    public let skipLint: Bool
```

Comment:
It's a bit confusing that one of the attributes here is to modify Tuist's behaviours further down the pipeline. I'd rather remove the listing all together. What do you think @fortmarek ?

Reply:
> remove the listing all together.

I am not sure what exactly you mean by that, could you maybe elaborate? I'd agree that the solution ain't perfect, but I couldn't think of a better way when I was implementing it

Reply:
I have deleted this as without the script this is no longer needed